hax/hax/motr/__init__.py

```diff
@@ -364,10 +370,12 @@ def _generate_sub_services(self,
             for x in service_list
         ]
         if notify_devices:
+            service_notes += self._generate_sub_disks(note, service_list, cns)
             # For process failure, we report failure for the corresponding
             # node (enclosure) and CVGs.
-            service_notes += self._generate_sub_disks(note, service_list, cns)
-            service_notes += self.notify_node_failure(note)
+            fid = Fid.from_struct(note.no_id)
+            if fid.container == ObjT.PROCESS.value:
```

Comment:
Let's move this to broadcast_ha_states() and invoke as follows,

```
if fid.container == ObjT.PROCESS.value and (st.status in (ServiceHealth.FAILED, ServiceHealth.OK):
    notes += self.notify_node_failure(note)
```

Reply:
Sure, could you please explain why should we notify node failure when ServiceHealth is **OK**?

Reply:
Done, we are node status. Hence updated the function name too.